multiclusterhub/charts/volsync-addon-controller/templates/volsync-addon-controller-clusterrolebinding.yaml

```diff
@@ -0,0 +1,22 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Values.org }}:{{ .Release.Name }}:clusterrolebinding
+  labels:
+    app: {{ template "volsync-addon-controller.name" . }}
+    chart: {{ template "volsync-addon-controller.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ template "volsync-addon-controller.name" . }}
+    helm.sh/chart: {{ template "volsync-addon-controller.chart" . }}
+    component: clusterrolebinding
+subjects:
+- kind: ServiceAccount
+  name: {{ template "volsync-addon-controller.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  kind: ClusterRole
+  name: {{ .Values.org }}:{{ .Release.Name }}:clusterrole
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
Missing newline at end of last line.

https://stackoverflow.com/a/729795/1437822